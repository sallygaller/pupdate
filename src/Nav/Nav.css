.Nav {
    text-align: center;
    display: flex;
  }
  
.Nav a,
.Nav a:active,
.Nav a:focus,
.Nav a:visited {
    color: #ececea;
    text-decoration: none;
}
  
.Nav a:hover {
    color: #ffffff;
    text-decoration: solid;
}
  
.Nav a:not(:last-of-type) {
    margin-right: 32px;
}