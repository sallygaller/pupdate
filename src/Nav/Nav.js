import React from "react";
import { Link } from "react-router-dom";
import "./Nav.css";

export default function Nav() {
  return (
    <nav className="Nav">
      <Link to="/availablepupdates">Available Pupdates</Link>{" "}
      <Link to="/pupdates">My Pupdates</Link>{" "}
      <Link to="/new-pupdate">New Pupdate</Link> <Link to="/pups">My Pups</Link>
      <Link to="/register">Register</Link>
      <Link to="/login">Log In</Link>
    </nav>
  );
}
