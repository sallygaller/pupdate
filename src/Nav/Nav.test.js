import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import Nav from "./Nav";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(
    <BrowserRouter>
      <Nav />
    </BrowserRouter>,
    div
  );
  ReactDOM.unmountComponentAtNode(div);
});

it("renders the UI as expected", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Nav />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
