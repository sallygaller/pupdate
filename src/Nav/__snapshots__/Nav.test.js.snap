// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<nav
  className="Nav"
>
  <div>
    <a
      aria-current={null}
      href="/login"
      onClick={[Function]}
    >
      Log in
    </a>
    <a
      aria-current={null}
      href="/register"
      onClick={[Function]}
    >
      Register
    </a>
  </div>
</nav>
`;
